libs/ledgerjs/packages/hw-app-btc/src/BtcNew.ts

```diff
@@ -474,7 +474,7 @@ function isPathNormal(path: string): boolean {
     pathElems.length >= 3 &&
     pathElems.length <= 5 &&
     [44 + h, 49 + h, 84 + h, 86 + h].some(v => v == pathElems[0]) &&
-    [0 + h, 1 + h].some(v => v == pathElems[1]) &&
+    [0 + h, 1 + h, 88 + h].some(v => v == pathElems[1]) &&
```

Comment:
I suggest putting the supported coin IDs (0, 1, 88) in a dedicated static array with an explicit name, and use this here, to make it slightly easier to understand for future maintainers.

Not saying the rest of `hw-app-btc` is perfectly clear and maintainable... but let's do what we can. :slightly_smiling_face:

Reply:
Additional suggestion: this array could already contain the [SLIP-44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) coin id of all the bitcoin family chains we support today, so that we don't have to change it again next time.

Reply:
Wow you're right :open_mouth: I didn't check Qtum in SLIP-44 but it's not 88 indeed...
Ok so forget about my 2nd suggestion, but let's create a const array with the proper values :+1:

Reply:
Done here 90e808dc39fc7fbf583f6bb75d76233eddd86c0f
Plus added tests on this function here aebe24ca2d012ed05a3ffb84fbd6d6cd7f60f91f before updating it to prevent issues